backend/src/main/java/eventlistener/security/model/AppUserDTO.java

```diff
@@ -0,0 +1,19 @@
+package eventlistener.security.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Document("appusers")
```

Comment:
To my understanding the idea of a dto is that it doesn't communicate directly with the db, but gets mapped to an object whose sole responsibility is  the communication with the db

Reply:
Renamed it because Sonar suggestion. I guess it should be fine.